src/helpers/parser.ts

```diff
@@ -25,19 +29,52 @@ export const gitIssueParser = async ({ owner, repo, issue_number }: GitParser):
   }
 };
 
-export const gitLinkedIssueParser = async ({ owner, repo, issue_number }: GitParser): Promise<string> => {
+export const gitLinkedIssueParser = async ({ owner, repo, pull_number }: GitParser): Promise<string> => {
   try {
-    const { data } = await axios.get(`https://github.com/${owner}/${repo}/pull/${issue_number}`);
+    const { data } = await axios.get(`https://github.com/${owner}/${repo}/pull/${pull_number}`);
     const dom = parse(data);
     const devForm = dom.querySelector("[data-target='create-branch.developmentForm']") as HTMLElement;
-    const linkedPRs = devForm.querySelectorAll(".my-1");
+    const linkedIssues = devForm.querySelectorAll(".my-1");
 
-    if (linkedPRs.length === 0) {
+    if (linkedIssues.length === 0) {
       return "";
     }
 
-    const prUrl = linkedPRs[0].querySelector("a")?.attrs?.href || "";
-    return prUrl;
+    const issueUrl = linkedIssues[0].querySelector("a")?.attrs?.href || "";
+    return issueUrl;
+  } catch (error) {
+    return "";
+  }
+};
+
+export const gitLinkedPrParser = async ({ owner, repo, issue_number }: GitParser) => {
+  try {
+    const { data } = await axios.get(`https://github.com/${owner}/${repo}/issues/${issue_number}`);
+    const context = getBotContext();
+    const payload = context.payload as Payload;
+    const dom = parse(data);
+    const devForm = dom.querySelector("[data-target='create-branch.developmentForm']") as HTMLElement;
+    const linkedPRs = devForm.querySelectorAll(".my-1");
```

Comment:
I wonder if we should 

1. Store all DOM selectors in our environment variables so that they can be updated quickly
2. Have scraper unit tests powered by ChatGPT that check if the selectors have changed for these elements.

Reply:
I also don't like hardcoded DOM selectors here because we should update the codebase if it changes. but i think we can do it later after doing some research about the better way.